trl/trainer/ppo_trainer.py

```diff
@@ -648,7 +649,9 @@ def log_stats(
                 warnings.warn(
                     "The game logs will not be logged because the batch does not contain the keys 'query' and 'response'."
                 )
-            elif self.config.log_with_wandb:
+            elif self.config.log_with == "wandb":
+                import wandb
+
                 table_rows = [list(r) for r in zip(batch["query"], batch["response"], rewards.cpu().tolist())]
                 wandb_logs.update({"game_log": wandb.Table(columns=["query", "response", "reward"], rows=table_rows)})
```

Comment:
Let's keep this as it's specific to `wandb`? wdyt?

Reply:
sounds good!